remix.init/index.js

```diff
@@ -235,6 +235,7 @@ const main = async ({ isTypeScript, packageManager, rootDirectory }) => {
     fs.rm(path.join(rootDirectory, ".github", "ISSUE_TEMPLATE"), {
       recursive: true,
     }),
+    fs.rm(path.join(rootDirectory, ".github", "dependabot.yml")),
```

Comment:
I wasn't sure if we wanted to keep this in the actual app

For me personally, I think this is valuable to have (same for normal dependencies btw), but not sure if we want to push this to every app

Reply:
we could always ask if they want it during remix init 🤷‍♂️. i'm on the fence

Reply:
Since stacks are already a bit opinionated, I think we should either remove it or keep it, but we never asked for anything in our official stack (yet).

Reply:
for sure, I'm on the fence about having it included by default, mostly because they're not testable by ci like regular dependencies would be

Reply:
We only have 1 workflow, which is ran on each PR, so as a matter of fact we _do_ test changes to the file.
With the current PR, we remove the Dependabot file just like we remove the PR template.

Reply:
> With the current PR, we remove the Dependabot file just like we remove the PR template.

yep, was just giving my reasons to keep that :)

> We only have 1 workflow, which is ran on each PR, so as a matter of fact we do test changes to the file.

ah right, duh 🤪